dvc/command/plot.py

```diff
@@ -0,0 +1,120 @@
+import argparse
+import logging
+
+from dvc.command.base import append_doc_link, CmdBase, fix_subparsers
+from dvc.exceptions import DvcException
+from dvc.utils import format_link
+
+logger = logging.getLogger(__name__)
+
+
+class CmdPlotShow(CmdBase):
+    def run(self):
+        try:
+            self.repo.plot(
+                datafile=self.args.datafile,
+                template=self.args.template,
+                file=self.args.file,
+            )
+
+        except DvcException:
+            logger.exception("plot show:")
```

Comment:
are you sure you need `:`? Logger will format it by itself, we don't use `:` anyehwere else.